docs/timeline/index.html

```diff
@@ -1021,6 +1021,14 @@ <h2 id="Configuration_Options">Configuration Options</h2>
       <td>A template function used to generate the contents of the items. The function is called by the Timeline with an items data as the first argument and the item element as the second, and must return HTML code as result. When the option template is specified, the items do not need to have a field <code>content</code>. See section <a href="#Templates">Templates</a> for a detailed explanation.</td>
     </tr>
 
+    <tr>
+      <td>visibleFrameTemplate</td>
+      <td>function</td>
+      <td>none</td>
+      <td>A template function used to generate the visible frame of the items. The function is called by the Timeline with an items data as the first argument and the item frame element as the second, and must return HTML code as result. When the option template is specified, the items do not need to have a field <code>content</code>. See section <a href="#Templates">Templates</a> for a detailed explanation.
+      This would be used as an additional way to add content that is constant in size with the visible frame of the item and does not get visibly hidden with the item's internal container: <code>vis-item-overflow</code> which is <code>overflow:hidden</code>. [see <a href="https://github.com/almende/vis/issues/1126">example</a>]</td>
```

Comment:
Do we really want to link to issues in the documentation? Maybe we better link to an example showing what the function is useful for?